NitroxServer/ConsoleCommands/FastCommand.cs

```diff
@@ -0,0 +1,74 @@
+using NitroxModel.DataStructures.GameLogic;
+using NitroxModel.Packets;
+using NitroxServer.ConsoleCommands.Abstract;
+using NitroxServer.ConsoleCommands.Abstract.Type;
+using NitroxServer.GameLogic;
+
+namespace NitroxServer.ConsoleCommands;
+
+public class FastCommand : Command
+{
+    private readonly PlayerManager playerManager;
+    private readonly SessionSettings sessionSettings;
+
+    public FastCommand(PlayerManager playerManager, SessionSettings sessionSettings) : base("fast", Perms.MODERATOR, "Enables/disables a fast cheat command, whether it be \"hatch\" or \"grow\"")
+    {
+        AddParameter(new TypeString("cheat", true, "The name of the fast cheat to change: \"hatch\" or \"grow\""));
```

Comment:
Would it make sense to make it an enum? If we rename FastCheatChanged.FastCheat we can even use it.